src/components/shared/PanelSearch.js

```diff
@@ -43,7 +44,7 @@ export class PanelSearch extends React.PureComponent<Props, State> {
     return (
       <div className={classNames('panelSearchField', className)}>
         <label className="panelSearchFieldLabel">
-          {label}
+          {label + ' '}
```

Comment:
This change was required because we don't have a space in the ftl texts anymore. I'm not super happy with this and ideally we should give this spacing in the css rather than as a space here. But the bottom panel relies on these types of spacings a lot. And this requires change in a bunch of places. I would prefer to do that in another PR.